src/main/java/seedu/storemando/commons/core/Messages.java

```diff
@@ -10,4 +10,5 @@
     public static final String MESSAGE_INVALID_ITEM_DISPLAYED_INDEX = "The item index provided is invalid";
     public static final String MESSAGE_MORE_THAN_ONE_ITEM_LISTED_OVERVIEW = "%1$d items listed!";
     public static final String MESSAGE_LESS_THAN_TWO_ITEMS_LISTED_OVERVIEW = "%1$d item listed!";
+    public static final String MESSAGE_NO_ITEM_IN_LIST = "There is no item in the list";
```

Comment:
Should we make it more explicit like this "There are no items in the current list"?

Reply:
Will update!